src/MigrationTools/_EngineV1/Configuration/Processing/WorkItemMigrationConfig.cs

```diff
@@ -80,6 +85,7 @@ public WorkItemMigrationConfig()
             AreaMaps = new Dictionary<string, string>();
             IterationMaps = new Dictionary<string, string>();
             MaxGracefulFailures = 0;
+            SkipRevisionWithInvalidIterationPath = false;
```

Comment:
have you checked with a config that does not have this field at all that they get false as default?

Reply:
Yep, this defaults to false.